backend/tracim_backend/views/core_api/user_controller.py

```diff
@@ -574,7 +598,7 @@ def last_active_content(self, context, request: TracimRequest, hapic_data=None):
         """
         app_config = request.registry.settings["CFG"]  # type: CFG
```

Comment:
Note: i'm not sure but if i remembers well there is 2 endpoints which are deprecated by the new ui and last_active_content is one of them .